data/remnant/remnant missions.txt

```diff
@@ -2798,12 +2798,14 @@ mission "Remnant: Cognizance 1"
 				"Korath Chaser" 2
 	on enter "Nenia"
 		dialog
-			`As you scan the system, the first thing you notice is that the void sprites' behavior seems different: There are fewer of them flying around, and the few that you see seem to be almost timid - floating quietly, and disappearing into the clouds whenever your ship moves towards them.`
-			`	After several minutes of recording the elusive void sprites, you decide to visit the next gas giant, recalling that Plume will want footage from both planets.`
-	on stopover
-		conversation
-			`Here you find the same behavior as before: void sprites have suddenly become timid and prone to diving into the shrouded depths of their world at the slightest quick movement.`
-			`	Despite the new challenge, you manage to record a variety of different void sprites before deciding to head back to <destination>.`
+			`As you scan the system, the first thing you notice is that the void sprites' behavior seems different: There are fewer of them flying around, and the few that you see seem to be almost timid - floating quietly, and dropping down into the cloud deck whenever your ship moves towards them. You recall that Plume asked you to visit both planets.`
+	on stopover # "Slylandro"
```

Comment:
Just dropping something here since this is the first time it's used.
Do the hashtags belong here? From my own (basic) scripting knowledge, hashtags _usually_ invalidate the things on the line that come after it. And if the intent here is simply to invalidate, what would be the point of adding hashtags instead of entirely removing the lines?
If this is something related to scripting that makes sense and this comment is wrong/irrelevant feel free to ignore. Just wanted to note it.

Reply:
I commented them out with the idea that - when support for multiple `on stopover` fields are added - these can be removed and the writing thus re-added to the game.

And yes, the intent is that the lines remain present, but not in use.

Reply:
Granted, I'm not sure if the Devs will be OK with doing that, in which case I'll delete all the commented stuff, but that's the idea anyway.